modules/node-tuning-operator.adoc

```diff
@@ -3,9 +3,19 @@
 // * scalability_and_performance/using-node-tuning-operator.adoc
 
 [id='about-node-tuning-operator-{context}']
-= Understanding the node tuning operator
+= About the node tuning Operator
 
-The node tuning operator helps you manage node-level tuning by orchestrating the
+The node tuning Operator helps you manage node-level tuning by orchestrating the
 tuned daemon.
 
-Here are more details about why it is used...
+The majority of high-performance applications require some level of kernel
+tuning. The node tuning Operator provides a unified management interface to
+users of node-level sysctls and more flexibility to add custom tuning specified
+by user needs. The Operator manages the containerized tuned daemon for
+{product-title} as a Kubernetes DaemonSet. It ensures custom tuning
+specification is passed to all containerized tuned daemons running in the
+cluster in the format that the daemons understand. The daemons run on all nodes
+in the cluster, one per node.
+
+In {product-title} 4.0 and later, the node tuning Operator is automatically
+deployed.
```

Comment:
Would the majority of user know what this feature does? What is the user trying to do? Is the point to match profiles or prevent non-matched profiles?

Reply:
The sentence "is automatically deployed" sounds a bit strange to me.  It is automatically deployed by the cluster version operator (CVO).  Or I'd indicate that the user doesn't have to do anything to get it deployed as it ships as part of the standard installation and the user doesn't have to do anything to have it running.

The point of the operator for 4.0 is to:
1) have a basic out-of-the box system tuning in the OCP cluster
2) make sure the applications shipped with 4.0 which have different tuning needs that the OS defaults run without issues
3) give the user a chance to specify their own tuning for their custom applications (this is a technology preview)